lib/generators/sincerely/aws_ses_events_generator.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails/generators'
+require 'rails/generators/active_record'
+
+module Sincerely
+  module Generators
+    class AwsSesEventsGenerator < ::Rails::Generators::Base
```

Comment:
but that is the point it should be a general `events` table that can handle all events not just aws ses events, so its name should be `EventsGenerator`